cvat/apps/engine/serializers.py

```diff
@@ -1185,6 +1195,9 @@ def create(self, validated_data):
 
         LabelSerializer.create_labels(labels, parent_instance=db_task)
 
+        if validated_data.get('assignee_id'):
+            db_task.assignee_updated_date = db_task.updated_date
+
         db_task.save()
```

Comment:
I guess that leads to different times in final `db_task.updated_date` and `db_task.assignee_updated_date`. Looks like we do not need to call `db_task.save()` here since db_task was created in a few lines above and was not updated before your changes. So, just moving `db_task.save(update_fields=["assignee_updated_date"])` to the condition block solves the issue.

Reply:
Agree, updated